### PR TITLE
Add support for the Inactive state

### PR DESCRIPTION
**Pull request by thgreasi**
Resolves: #13
Change-type: minor
Signed-off-by: Thodoris Greasidis <thodoris@resin.io>

### CODE REVIEW

File: build/status.js [hunk 1]
```diff
@@ -91,14 +95,20 @@ exports.statuses = [
  */
 
 exports.getStatus = function(device) {
-  var install, installs, lastSeenDate, ref, serviceName;
+  var install, installs, lastSeenDate, neverSeen, ref, serviceName;
```
Comment (CameronDiver) [inline build/status.js:98]: I'd prefer this to be added to another commit, but don't care all that much.

**Review by CameronDiver — APPROVED** [review]

**Review by pimterry — APPROVED** [review]

**Comment by thgreasi** [comment]
@resin-ci retest
